src/redux/reducers/stock/stockReducer.test.js

```diff
@@ -32,14 +32,14 @@ describe('Action creators', () => {
   });
 });
 
+const sampleResponse = [{ symbol: 'btc', image: 'https://assets.coingecko.com/coins/images/1/large/bitcoin.png?1547033579', current_price: 34883, market_cap: 653759402456, price_change_percentage_24h: 3.91425, market_cap_change_24h: 26894035129 }];
```

Comment:
Please define `https://eslint.org/docs/2.0.0/rules/max-len` rule with some value eg. 120 characters. After that we should break lines of structures like this one.